src/styles/blocks/stopWatch.scss

```diff
@@ -0,0 +1,46 @@
+.stopWatch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid black;
+  border-radius: $watchEdge;
+  height: 400px;
+  width: 400px;
+  &__smallCircle {
```

Comment:
better to make empty line between different sets of rules for increasing readability

Reply:
will consider this in the future tasks!